spec/views/verify/review/new.html.slim_spec.rb

```diff
@@ -7,7 +7,7 @@
     before do
       user = build_stubbed(:user, :signed_up)
       allow(view).to receive(:current_user).and_return(user)
-      allow(view).to receive(:idv_params).and_return(
+      @idv_params = {
```

Comment:
I thought we had to use assigns, but since this works, and other tests do the same, not worth worrying about on this PR.